operational_analysis/methods/plant_analysis.py

```diff
@@ -919,20 +907,28 @@ def sample_long_term_reanalysis(self):
         return long_term_reg_inputs.copy()
 
     @logged_method_call
-    def sample_long_term_losses(self):
+    def sample_long_term_losses(self, gross_lt):
         """
         This function calculates long-term availability and curtailment losses based on the Monte Carlo sampled
-        historical availability and curtailment data
+        historical availability and curtailment data. To estimate long-term losses, average percentage monthly losses 
+        are weighted by monthly long-term gross energy.
         
         Args:
-            n(:obj:`integer`): The Monte Carlo iteration number
+            :obj:`pandas.Series`: Time series of long-term gross energy
         
```

Comment:
Again a minor thing, I've noticed that in some other functions in the code we use something like the following: gross_lt(:obj:`dataframe`) instead of :obj:`pandas.Series`. Is there a preferred convention and do we want to stick with that one?

Reply:
Yes, that does seem to be the most commonly used style for args. I changed it.